Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.psm1

```diff
@@ -124,19 +194,24 @@ function Get-FilteredProjectsToBuild($settings, $projects, $baseFolder, $modifie
             - project: The project to build
             - buildMode: The build mode to use
 #>
-function Get-ProjectsToBuild(
-    [Parameter(HelpMessage = "The folder to scan for projects to build", Mandatory = $true)]
-    $baseFolder,
-    [Parameter(HelpMessage = "An array of changed files paths, used to filter the projects to build", Mandatory = $false)]
-    $modifiedFiles = @(),
-    [Parameter(HelpMessage = "The maximum depth to build the dependency tree", Mandatory = $false)]
-    $maxBuildDepth = 0
-)
-{
+function Get-ProjectsToBuild {
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'modifiedFiles', Justification = 'False-positive. Used in a Where-Object filter')]
```

Comment:
Yeah, but then the convenient PS syntax is lost :/

How about this hack:
Instead of this:
```
... | Where-Object {... $param }
```
Use this:
```
$var = $param
... | Where-Object {... $var}
```

Reply:
The buildDimensions array in the function above is built without piping, I think that is nice code.

Reply:
Fixed.